Library/Formula/validns.rb

```diff
@@ -0,0 +1,30 @@
+class Validns < Formula
+  homepage "http://www.validns.net/"
+  url "http://www.validns.net/download/validns-0.8.tar.gz"
+  sha256 "df2db0eaa998a0411ff4c1c4e417eb82d32aec4835f92f45f26c66c8d1d5bd22"
+
+  depends_on 'judy'
+
+  patch :DATA
+
+  def install
+    system "make"
+    bin.install 'validns'
+    man1.install 'validns.1'
+  end
+
+  test do
+    system "#{bin}/validns -v 2>&1 | grep 'quit on first validation error'"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!